.travis.yml

```diff
@@ -70,8 +70,12 @@ matrix:
       env: PROJECT_DIR=examples/acf TOOLCHAIN=libcxx
     - os: osx
       env: PROJECT_DIR=examples/acf TOOLCHAIN=osx-10-12
+    # FIXME: Unknown error in testing (can't get meaningful log)
+    #- os: osx
+    #  env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-10-3 VERBOSE=0
     - os: osx
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-10-3
+      osx_image: xcode9.2
+      env: PROJECT_DIR=examples/OpenCV TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-arm64 VERBOSE=0
```

Comment:
Please add extra toolchains after the main set and wrap it with `# Extra {` / `}`, example:
* https://github.com/ingenue/hunter/blob/dc51b206d84c4eca0e9446b6fac6d8a3da104b27/.travis.yml#L134-L157